package.json

```diff
@@ -64,7 +64,7 @@
     "app-module-path": "^1.0.5",
     "babel-cli": "^6.3.17",
     "babel-core": "^6.3.21",
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "5.0.0",
```

Comment:
Otherwise it will try to install 5.0.1 which as a peerDep eslint <2.0.0, which goes in conflict with material-ui dev dependency